pages/docs/guides/push-notifications-firebase.md

```diff
@@ -385,7 +385,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 If you would like to receive the Firebase FCM token from iOS instead of the raw APNS token, you will also need to add two new imports at the top of the file:
 
 ```swift
-import FirebaseInstanceID // Add this line after import FirebaseCore
+import FirebaseInstallations // Add this line after import FirebaseCore
```

Comment:
Is this needed? The included code only references the `Messaging` module, not the `Installations` module.

Reply:
Hmm, I can try it that way; I was going off the link (https://firebase.google.com/docs/projects/manage-installations#retrieving-an-fcm-registration-token) and I may have mistakenly assumed I needed to import FirebaseInstallations

Reply:
Okay, confirmed you don’t need this import. I’ll update the PR.